website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -296,7 +310,29 @@ api.approveTask = {
       scoreTask: task,
     });
 
-    await Bluebird.all([assignedUser.save(), task.save()]);
+    let managerIds = Object.keys(group.managers);
+    managerIds.push(group.leader);
+    let managers = await User.find({_id: managerIds}).exec(); // Use this method so we can get access to notifications
+
+    let managerPromises = [];
+    managers.forEach((manager) => {
+      let notificationIndex =  findIndex(manager.notifications, function findNotification (notification) {
+        let notificationMessage = res.t('userHasRequestedTaskApproval', {
```

Comment:
there isn't a better way to identify the notification? Relaying on the translated text doesn't seem optimal and would fail if the user changes its language